changelog.txt

```diff
@@ -24,10 +30,6 @@ Version 3.2.0
 - [MINOR] Device PoP keys are now generated with attestation flags and expose a certificate chain getter (#1247)
 - [PATCH] Fix MSAL-CPP injecting javascript loop (#1238)
 - [MINOR] Adds new API to support ADAL/MSAL migration off of AuthenticationSettings#setSecretKey to a 'managed key' generated by Common (#1244)
-- [PATCH] Improved performance of getAccounts() call by reducing number of cache round-trips (#1271)
```

Comment:
Why did these changes move from 3.2.0 to 3.3.1?

Reply:
They shouldn't be part of 3.2.0, they should be in 3.3.1, see https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1313#discussion_r614217977